service/src/main/java/uk/gov/hmcts/reform/fpl/service/cmo/UploadCMOService.java

```diff
@@ -73,34 +72,13 @@ public UploadCMOEventData getInitialPageData(CaseData caseData) {
     @Deprecated
     public UploadCMOEventData getInitialPageData(List<Element<HearingBooking>> pastHearings,
```

Comment:
This whole method can be deleted if the new cmo stuff that I did the other week has been toggled on